models/junos/static_routes/junos_static_routes.yaml

```diff
@@ -0,0 +1,61 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'network'
+  }
+
+NETWORK_OS: junos
+RESOURCE: static_routes
+COPYRIGHT: Copyright 2019 Red Hat
+LICENSE: gpl-3.0.txt
+
+DOCUMENTATION: |
+  module: junos_static_routes
+  version_added: 2.10
+  short_description: Manage static routes on Juniper JUNOS devices
+  description: This module provides declarative management of static routes on Juniper JUNOS devices
+  author: Daniel Mellado (@dmellado)
+  requirements:
+    - ncclient (>=v0.6.4)
+  notes:
+    - This module requires the netconf system service be enabled on the device being managed.
+    - This module works with connection C(netconf). See L(the Junos OS Platform Options,../network/user_guide/platform_junos.html).
+    - Tested against JunOS v18.4R1
+  options:
+    config:
+      description: A dictionary of static routes options
+      type: list
+      elements: dict
+      suboptions:
```

Comment:
Since we have the ability to specify the VRF and Address Family for static routes in IOS-XR, we had to add a top-level key called `routes` under which we have the static route specific parameters.

Reply:
I can just add it there for consistency